data/mods/blazemod/blaze_blob_parts.json

```diff
@@ -3153,12 +1694,8 @@
     "damage_modifier": 10,
     "folded_volume": 1,
     "breaks_into": [ { "item": "stickiegrow" } ],
-    "requirements": {
-      "repair": {
-        "time": 30000,
-        "using": [ [ "blobrep", 1 ] ]
-      }
-    },
+    "description": "A living blob turned into a vehicle weapon.",
```

Comment:
Ok, the beginning had mostly `heavy vehicle weapon` and lines on 1617, 1637, 1657, 1677, 1697, 1717 and 1737 are "vehicle weapon" so is there supposed to be a difference?

Reply:
I did this over 5 days. I might have gotten inconsistent without noticing.

Reply:
Inconsistencies fixed, thanks for the review.